bigquery_etl/view/__init__.py

```diff
@@ -254,6 +254,12 @@ def has_changes(self, target_project=None):
         expected_view_query = CREATE_VIEW_PATTERN.sub(
             "", sqlparse.format(self.content, strip_comments=True), count=1
         ).strip(";" + string.whitespace)
+        
+        if table.view_query is None:
+            # Materialized views do not have view_query.
+            # Always assume they are unchanged since they can't be replaced anyway
+            return False
```

Comment:
This kind of special casing makes the codebase hard to reason about. Is there a way for us to prevent `bqetl query publish` from indexing materialized views to begin with?